container_linux_test.go

```diff
@@ -380,3 +380,137 @@ func TestContainerAttach(t *testing.T) {
 		t.Errorf("expected output %q but received %q", expected, output)
 	}
 }
+
+func TestContainerAttachProcess(t *testing.T) {
```

Comment:
This doesn't work on Windows?

If it doesn't, if it doesn't maybe open an issue, I can change the impl to support it. There's no reason for it not to work in a case where containerd hasn't restarted (since we kill all running process on restart)